cli/embedded-files/eirini/quarks-secrets.yaml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
This file achieves the same results as this code: https://github.com/SUSE/carrier/blob/main/eirini/eirini-release/deploy/scripts/generate_eirini_tls.sh#L14-L23